### PR TITLE
Feature/tool tip text rebase

### PR DESCRIPTION
**Pull request by hectoruch**

### CODE REVIEW

File: public/definitions.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
...
...ically to identify “apparent fishing effort.” As a result, it is possible that some fishing effort is not identified as such by Global Fishing Watch; conversely, Global Fishing Watch may show apparent fishing effort where fishing is not actually taking place. For these reasons, Global Fishing Watch qualifies designations of vessel fishing effort, including synonyms of the term “fishing effort,” such as “fishing” or “fishing activity,” as “apparent,” rather than certain. Any/all Global Fishing Watch information about “apparent fishing effort” should be considered an estimate and must be relied upon solely at your own risk. Global Fishing Watch is taking steps to make sure fishing effort designations are as accurate as possible. Global Fishing Watch fishing detection algorithms are developed and tested using actual fishing event data collected by observers, combined with expert analysis of vessel movement data resulting in the manual classification of thousands of known fishing events. Global Fishing Watch also collaborates extensively with academic researchers through our research partner program to share fishing effort classification data and automated classification technique</p>"
```
Comment (tiagojsag) [inline public/definitions.json]: why did you change this?

File: app/src/components/Home/InfoWeb.jsx [hunk 2]
```diff
@@ -27,7 +28,11 @@ class InfoWeb extends Component {
             </p>
             <h3>STEP 3: YOU</h3>
             <p>
-              Once the data is visualized on Global Fishing Watch, anyone can track fishing activity across the globe.
```
Comment (tiagojsag) [inline app/src/components/Home/InfoWeb.jsx]: missing text

File: app/src/components/Shared/ToolTip.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import ToolTipStyle from '../../../styles/components/c-tooltip-info.scss';
+import ImgWhite from '../../../assets/icons/info_white.svg';
+import ImgBlack from '../../../assets/icons/info_black.svg';
+
+class ToolTip extends Component {
+
+  render() {
+    return (
+      <Link
+        to="/definitions"
```
Comment (tiagojsag) [inline app/src/components/Shared/ToolTip.jsx]: https://www.pivotaltracker.com/story/show/128939267

When the user clicks this, they need to be taken to the "definitions" page, with "fishing effort" open and autoscroll to it.

Comment (andresgnlez) [inline app/src/components/Shared/ToolTip.jsx]: I think that behavior has nothing to do with tooltip implementation. I know in Pivotal the task relates both things, but please, separate them. There's no connection between them. That behavior will develop in other branch, not here.

Comment (tiagojsag) [inline app/src/components/Shared/ToolTip.jsx]: You've got a point. new task created

@hectoruch please fix all the other comments here ASAP
@andresgnlez I put the new task at the top of the priority list, please pick it up once you finish your current task

Comment (andresgnlez) [inline app/src/components/Shared/ToolTip.jsx]: @tiagojsag roger!

File: public/definitions.json (hunk 1 above)
Comment (hectoruch) [inline public/definitions.json]: On pivotal task, say that is neccesary to change it

**Comment by hectoruch** [comment]
Move to https://github.com/Vizzuality/GlobalFishingWatch/pull/192
